### PR TITLE
Added students sample student data

### PR DESCRIPTION
**Pull request by jchin01**

### CODE REVIEW

**Comment by chanlawrencet** [comment]
Updated with syntax fixes

File: bin/resetDatabase.py [hunk 1]
```diff
@@ -4,10 +4,13 @@
...
     users = db.users
     print('Made users')
-
     for i in range(0, 10):
         initData = {
-            'name': 'user' + str(i),
+            'first_name': 'first' + str(i),
+            'middle_name': 'middle' + str(i),
+            'last_name': 'last' + str(i),
+            'DOB': str(i)*4 + str(i)*2 + str(i)*2,
```
Comment (chanlawrencet) [inline bin/resetDatabase.py]: This DOB will be `00000000` and `11111111`, etc...
is this what you want?

File: bin/resetDatabase.py [hunk 2]
```diff
@@ -4,10 +4,13 @@
 def makeUsers(db):
     users = db.users
```
Comment (chanlawrencet) [inline bin/resetDatabase.py]: Do you want to add to the `users` collection?

File: bin/resetDatabase.py [hunk 3]
```diff
@@ -4,10 +4,13 @@
 def makeUsers(db):
```
Comment (chanlawrencet) [inline bin/resetDatabase.py]: Also consider a different name for this function

File: bin/resetDatabase.py [hunk 4]
```diff
@@ -4,10 +4,13 @@
...
     users = db.users
     print('Made users')
-
     for i in range(0, 10):
         initData = {
```
Comment (chanlawrencet) [inline bin/resetDatabase.py]: I'm not sure if this follows the google doc examples (eg. where are forms?)

**Comment by jhuang09** [comment]
Hiya! I have made some additions/changes to the Students collection to reflect the design on the Google doc. I have also added the sample data for the Forms collection. Hopefully, each form from the sample Student collection will have a corresponding form in the Forms collection.

**Pull request merged**